exercises/react/tests/react.rs

```diff
@@ -101,32 +101,29 @@ fn error_setting_a_compute_cell() {
 
 use std::cell::RefCell;
 struct Expector {
-    have_value: RefCell<bool>,
-    value: RefCell<isize>,
+    value: RefCell<Option<isize>>,
 }
 
 impl Expector {
     fn new() -> Self {
         Expector {
-            have_value: RefCell::new(false),
-            value: RefCell::new(0),
+            value: RefCell::new(None),
         }
     }
 
     fn expect_to_have_been_called_with(&self, v: isize) {
-        assert!(*self.have_value.borrow(), "Callback was not called, but should have been");
-        assert_eq!(*self.value.borrow(), v, "Callback was called with incorrect value");
-        *self.have_value.borrow_mut() = false;
+        assert_ne!(*self.value.borrow(), None, "Callback was not called, but should have been");
```

Comment:
not necessary, given the below line, but maybe helpful for a better error message???

Reply:
Sure; to present a human-friendly error message in test code is definitely worth an extra line and a single repeated branch.